configs/components/cfpropertylist.json

```diff
@@ -0,0 +1 @@
+{"url": "git@github.com:ckruse/CFPropertyList.git", "ref": "2.2.7"}
```

Comment:
The rewrite engine does not support git yet, however, we should still add this to the mirror.